docs/content/design/coordinator.md

```diff
@@ -188,6 +188,15 @@ Returns all rules for a specified datasource and includes default datasource.
 
  Returns audit history of rules. default value of interval can be specified by setting `druid.audit.manager.auditHistoryMillis` (1 week if not configured) in coordinator runtime.properties
 
+#### Namespaced Lookups
+
+* `/druid/coordinator/v1/lookups`
```

Comment:
is this suppose to be  `/druid/coordinator/v1/namespaces` as in [code](https://github.com/metamx/druid/blob/queryTimeLookup/server/src/main/java/io/druid/server/http/NamespacesResource.java#L45-45)